docs/tutorials/using-conponents.md

```diff
@@ -0,0 +1,106 @@
+---
+Level: Beginner
+---
+# Using components
+The best option you should consider when you plan to use an ADF component and want to learn the details of its usage, is always to check the documentation for the component you are looking to use. More in general, there are three different ways to use, extend and configure an ADF component:
```

Comment:
This text is good but the first paragraph of the file gets used for the brief description that appears in the index page. I think this paragraph might be a bit long - maybe add a summary of one or two sentences for the first paragraph and then continue with this part.

Reply:
Done... if I understand correctly the note.
Please let me know if you suggested something different.